src/parser/core/modules/Weaving.js

```diff
@@ -35,6 +35,13 @@ export default class Weaving extends Module {
 
 	static title = t('core.weaving.title')`Weaving Issues`
 
+	// WVR Focused synth lmao
+	_suggestionIcon = 'https://xivapi.com/i/001000/001785.png'
```

Comment:
only private properties should be prefixed with `_`, given these are "protected", can omit the prefix

Reply:
They've been protected - secret service isn't cheap though.